app/presenters/V1/nsm/letters_and_calls_costs_presenter.rb

```diff
@@ -9,6 +9,7 @@ def letters_and_calls_for_calculation
         [letters_for_calculation, calls_for_calculation]
       end
 
+      # TODO: Add real assessed costs if using caseworker assessed data
```

Comment:
Make this a ticket and link it. We have too many dead TODOs that never get actioned